README.md

```diff
@@ -4,6 +4,113 @@
 
 This library provides instrumentation for CDI modules in Java EE applications.
 
+## Usage
+
+For a concrete and comprehensive set of examples, refer to the `opentracing-cdi-example`
+module.
+
+### Injecting a tracer
+
+In your managed beans, you can get an instance of the appropriate tracer via:
+
+```java
+@Inject
+Tracer tracer;
+```
+
+If you have your own `Tracer` producer, you'll need to mark your own as an
+`@Alternative`
```

Comment:
`@Alternative` for CDI means "this is more important". See http://docs.oracle.com/javaee/6/tutorial/doc/gjsdf.html